.github/workflows/on-pull-request.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  pull_request:
```

Comment:
CONSIDER: Can we make this run on merge to main or prevent that?

Reply:
It's just a reply